tests/unit/cluster/base.tcl

```diff
@@ -0,0 +1,149 @@
+# Check the basic monitoring and failover capabilities.
+
+# make sure the test infra won't use SELECT
+set old_singledb $::singledb
+set ::singledb 1
+
+tags {tls:skip external:skip cluster} {
+
+set base_conf [list cluster-enabled yes]
+start_multiple_servers 5 [list overrides $base_conf] {
+
+test "Cluster nodes are reachable" {
+    for {set id 0} {$id < [llength $::servers]} {incr id} {
+        # Every node should be reachable.
+        wait_for_condition 1000 50 {
+            ([catch {R $id ping} ping_reply] == 0) &&
+            ($ping_reply eq {PONG})
+        } else {
+            catch {R $id ping} err
+            fail "Node #$id keeps replying '$err' to PING."
+        }
+    }
+}
+
+test "Cluster nodes hard reset" {
```

Comment:
Did you pull this in because of the other tests in this file were failing?

Reply:
no, this is actually because I want to cover the "Cluster nodes hard reset" test in init-tests.tcl. do you want me to remove it? the base.tcl can still pass without the hard reset

Reply:
I removed it, it seemed unnecessary